examples/labeling/point-aggregation-cluster-count.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Exported map | CARTO VL</title>
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<script src="../../dist/carto-vl.js"></script>
+<script src="https://api.tiles.mapbox.com/mapbox-gl-js/v0.52.0/mapbox-gl.js"></script>
+<link href="https://api.tiles.mapbox.com/mapbox-gl-js/v0.52.0/mapbox-gl.css" rel="stylesheet" />
+<link rel="stylesheet" type="text/css" href="../style.css">
+<style>
+    html, body {
+        margin: 0;
+    }
+    #map {
+        position: absolute;
+        width: 100%;
+        height: 100%;
+    }
+</style>
+</head>
+<body>
+<div id="map"></div>
+<script>
+    const map = new mapboxgl.Map({
+        container: 'map',
+        style: 'https://basemaps.cartocdn.com/gl/dark-matter-gl-style/style.json',
```

Comment:
We can now use carto.basemap enum for the style, eg. carto.basemaps.voyager

Reply:
You're right! I've copy-pasted other example, so I'll change it there as well.